lib/binary_parser.ts

```diff
@@ -325,25 +325,15 @@ export class Parser<O = {}> {
     return (type + this.endian.toLowerCase()) as PrimitiveTypes;
   }
 
-  uint8(): Next<O, unknown, number>;
   uint8<N extends string, F = number>(
     varName: N,
-    options?: ParserOptions<O, number, F>
-  ): Next<O, N, F>;
-  uint8<N extends string, F = number>(
-    varName?: N,
     options: ParserOptions<O, number, F> = {}
   ): Next<O, N, F> {
     return this.primitiveN("uint8", varName, options);
   }
 
-  uint16(): Next<O, unknown, number>;
```

Comment:
There is no other way to get a `uint16` parser without explicitly specifying endianness. This is useful in choice:

```javascript
choices: {
  0: Parser.uint16(), // "uint16" doesn't work
  1: Parser.uint32(), // "uint32" doesn't work
}
```